### ISSUE TITLE
App stops wotking after sometime

### ISSUE THREAD

**Issue by noideastowork** [Reeywhaar/middleme#5]
Thank you so much for the app! It's really useful!

However, there is one issue: after a while, the app stops working properly. When I try to tap with three fingers, nothing happens. To fix this, I need to open the app again.

Is it possible to fix it?

MacBook Pro 2021, M1 Pro, 14.4 (23E214)

**Comment by reeywhaar** [comment]
Yeah, I'm experiencing same issue, but just can't get a time for this minor annoyance. Help appleciated!

**Comment by reeywhaar** [comment]
Think it's fixed in [11](https://github.com/Reeywhaar/middleme/releases/tag/1.11)
